src/test/SmartComponents/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -338,5 +337,56 @@ describe('Click through all steps', () => {
         await screen.findByText('rhel-8');
         await screen.findByText('aws');
         await screen.findByText('Register the system on first boot');
+
+        // mock the backend API
+        const composeImage = jest.spyOn(api, 'composeImage');
+        composeImage.mockImplementation(async () => ({ id: 'test-me' }));
+
+        const create = screen.getByRole('button', { name: /Create/ });
+        create.click();
+
+        // API request sent to backend
+        expect(composeImage).toHaveBeenCalled();
+
+        // returns back to the landing page
+        // await waitFor(
+        //    () => [
+        //        screen.getByText('Create a new image'),
+        //        screen.getByRole('button', { name: /Create image/ }),
+        //    ]
+        // );
```

Comment:
@Gundersanne in the source code I see `426: this.props.history.push('/landing');` but if we uncomment the wait above it fails, see https://travis-ci.com/github/osbuild/image-builder-frontend/builds/202712706. 

Also clicking the Create button in ci.cloud doesn't send me back to the landing page so I think it may be broken.

Reply:
Well the request needs to succeed before it sends you to the landing page, so if there's no api that's returning `201` with `{"id": uuid}` in  the response this is expected.

Reply:
Yup, that makes sense, but isn't the mock above returning what we need already ? From the coverage change report I can see execution traverses the lines after the mocked API returns successfully and the redirect on 426 seem to have been executed.